### PR TITLE
small cleanup in victor utils

### PR DESCRIPTION
**Pull request by PeterMitrano**
also remove hard-coded prefix slash in paths

### CODE REVIEW

**Comment by PeterMitrano** [comment]
all spacing changes are according to autoformatting, I would suggest reviewing with "ignore whitespace" checked

File: victor_hardware_interface/src/victor_hardware_interface/victor_utils.py [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
 
+
 class Stiffness(Enum):
     STIFF = 1
     MEDIUM = 3
-    SOFT  = 2
+    SOFT = 2
+
+
+def get_new_control_mode(control_mode: ControlMode, stiffness=Stiffness.MEDIUM, vel=0.1, accel=0.1):
```
Comment (bsaund) [inline victor_hardware_interface/src/victor_hardware_interface/victor_utils.py:16]: `get_control_mode_params` might be a better name

File: victor_hardware_interface/src/victor_hardware_interface/victor_utils.py [hunk 2]
```diff
@@ -127,11 +123,67 @@ def get_joint_impedance_params(stiffness, vel=0.1, accel=0.1):
 
     else:
         rospy.logerr("Unknown stiffness for Joint Impedance")
-        assert(False)
+        assert False
 
     return new_control_mode
 
 
+def get_cartesian_impedance_params(velocity=2.5):
```
Comment (bsaund) [inline victor_hardware_interface/src/victor_hardware_interface/victor_utils.py:131]: Where did these numbers come from? Have you tested these?

**Review by bsaund — APPROVED** [review]

File: victor_hardware_interface/src/victor_hardware_interface/victor_utils.py (hunk 2 above)
Comment (PeterMitrano) [reply victor_hardware_interface/src/victor_hardware_interface/victor_utils.py:131]: I got them from johnson, pretty sure he's tested them @LemonPi emon

**Review by bsaund — APPROVED** [review]

File: victor_hardware_interface/src/victor_hardware_interface/victor_utils.py (hunk 2 above)
Comment (LemonPi) [reply victor_hardware_interface/src/victor_hardware_interface/victor_utils.py:131]: Yup these are tested values, but you mentioned that velocity=2.5 as default may be too low? The motions I've been doing are pretty small so it's been appropriate.

Comment (PeterMitrano) [reply victor_hardware_interface/src/victor_hardware_interface/victor_utils.py:131]: we can leave it at this, slow by default is better than fast

**Pull request merged**